airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/IdentifierNamingResolvable.java

```diff
@@ -22,9 +22,9 @@
  * SOFTWARE.
  */
 
-package io.airbyte.integrations.base;
+package io.airbyte.integrations.destination;
 
-public interface SQLNamingResolvable {
+public interface IdentifierNamingResolvable {
```

Comment:
Comments still mention SQL

Reply:
Can you explain what "IdentifierNamingResolvable" means? I don't get what action `Resolvable` ties to

Should it be `NameTransformer` or smthing like that?

Reply:
The methods mention `Table` & `RawTable` which sound SQL specific

Reply:
I marked them as @Deprecated because if I can manage to refactor all destinations to use the `TmpDestinationConsumer` class (that is handling renaming output tables with a proper suffix to the table names), then we won't need these methods anymore...